tennis.py

```diff
@@ -13,7 +12,12 @@ def __init__(self, player1Name, player2Name):
         self.player2 = Player(player2Name)
 
     def won_point(self, playerName):
-        if playerName == self.player1.name:
+        if self.score().startswith("Win for"):
+            raise Exception('Game is over')
+
+        if playerName != self.player1.name and playerName != self.player2.name:
+            raise Exception(playerName + " is not playing")
+        elif playerName == self.player1.name:
```

Comment:
```suggestion
        if playerName == self.player1.name:
```
Le `if` est facultatif car l'exception quitte le programme (comme un return)